com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/textureset/MaxRectsLayoutStrategy.java

```diff
@@ -217,6 +229,21 @@ static int getExponentNextOrMatchingPowerOfTwo(int value) {
         return exponent;
     }
 
+    private boolean canFitInPage(int width, int height, RectStats stats, boolean fully) {
+        long availableArea = (long)width * (long)height;
+        if (fully && availableArea < stats.totalArea) {
+            return false;
+        }
+
+        if (!settings.rotation) {
```

Comment:
would it make sense moving the "settings.rotation" to an argument?
I.e. making this a function, and not a method?